### PR TITLE
Marco/cleanup commerciokyc

### PR DESCRIPTION
**Pull request by marcotradenet**
## Checklist
- [ ] Targeted PR against correct branch
- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For Admin Use:
- [ ] Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- [ ] Reviewers Assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]"

### CODE REVIEW

File: x/commerciokyc/keeper/grpc_query_invite.go [hunk 1]
```diff
@@ -3,13 +3,50 @@ package keeper
...
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 func (k Keeper) Invites(c context.Context, req *types.QueryInvitesRequest) (*types.QueryInvitesResponse, error) {
 	ctx := sdk.UnwrapSDKContext(c)
-	invites := k.GetInvites(ctx)
-	return &types.QueryInvitesResponse{Invites: invites}, nil
+	var invites []*types.Invite
```
Comment (fabtagliaferro) [inline x/commerciokyc/keeper/grpc_query_invite.go:18]: Consider using `invites := []*types.Invite{}`

File: x/commerciokyc/keeper/grpc_query_membership.go [hunk 2]
```diff
@@ -3,24 +3,48 @@ package keeper
...
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 func (k Keeper) Memberships(c context.Context, req *types.QueryMembershipsRequest) (*types.QueryMembershipsResponse, error) {
 	ctx := sdk.UnwrapSDKContext(c)
-	memberships := k.GetMemberships(ctx)
-	return &types.QueryMembershipsResponse{Memberships: memberships}, nil
+	var memberships []*types.Membership
```
Comment (fabtagliaferro) [inline x/commerciokyc/keeper/grpc_query_membership.go:18]: Consider using `memberships := []*types.Membership{}`

File: x/commerciokyc/keeper/grpc_query_membership.go [hunk 3]
```diff
@@ -3,24 +3,48 @@ package keeper
...
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkErr "github.com/cosmos/cosmos-sdk/types/errors"
 
 	"github.com/commercionetwork/commercionetwork/x/commerciokyc/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 func (k Keeper) Memberships(c context.Context, req *types.QueryMembershipsRequest) (*types.QueryMembershipsResponse, error) {
```
Comment (fabtagliaferro) [inline x/commerciokyc/keeper/grpc_query_membership.go:16]: The name of the method could mislead to think this returns all the memberships, but it returns only the valid ones. Consider renaming it (e.g. `ValidMemberships`) or to add documentation comments above the signature.

File: x/commerciokyc/keeper/querier.go [hunk 4]
```diff
@@ -58,6 +60,26 @@ func queryGetInvites(ctx sdk.Context, _ []string, keeper Keeper, legacyQuerierCd
 	return bz, nil
 }
 
+// queryGetMembership allows to retrieve the current membership of a user having a specified address
+func queryGetInvite(ctx sdk.Context, path []string, keeper Keeper, legacyQuerierCdc *codec.LegacyAmino) (res []byte, err error) {
```
Comment (fabtagliaferro) [inline x/commerciokyc/keeper/querier.go:64]: `res` and `err` named return variables are not used, consider removing them

File: x/commerciokyc/keeper/querier.go [hunk 5]
```diff
@@ -58,6 +60,26 @@ func queryGetInvites(ctx sdk.Context, _ []string, keeper Keeper, legacyQuerierCd
...
+	address, err2 := sdk.AccAddressFromBech32(path[0])
```
Comment (fabtagliaferro) [inline x/commerciokyc/keeper/querier.go:65]: After removing the named return variable `err` this name can be used instead of `err2`

File: x/commerciokyc/keeper/grpc_query_invite_test.go [hunk 6]
```diff
@@ -58,3 +67,54 @@ func (suite *KeeperTestSuite) TestGRPCInvites() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestGRPCInvite() {
```
Comment (fabtagliaferro) [inline x/commerciokyc/keeper/grpc_query_invite_test.go:71]: Consider adding a test case where the store is empty (also in `TestGRPCInvites()`)

**Review by fabtagliaferro — APPROVED** [review]
